distributed/utils.py

```diff
@@ -1543,3 +1543,27 @@ def clean_dashboard_address(addr, default_listen_ip=""):
         port = addr
 
     return {"address": host, "port": port}
+
+
+@functools.lru_cache(maxsize=256)
+def _clean_filename(filename):
+    """Cleans filename from sensitive information"""
+
+    if filename.startswith(sys.exec_prefix):
+        filename = filename.replace(sys.exec_prefix, "...")
+    elif filename.startswith(os.path.expanduser("~")):
+        filename = filename.replace(os.path.expanduser("~"), "")
+        filename = ".../" + "/".join(filename.split("/")[-3:])
+    else:
+        filename = ".../" + "/".join(filename.split("/")[-3:])
```

Comment:
Here, and in a few other places, we're using `/` as the path separator. This is true for POSIX systems, but not for Windows which uses `\\`. We should use [`os.sep`](https://docs.python.org/3/library/os.html#os.sep) instead of `/`